packages/flutter_tools/lib/src/flutter_manifest.dart

```diff
@@ -614,11 +608,18 @@ void _validateFlutter(YamlMap? yaml, List<String> errors) {
       case 'deferred-components':
         _validateDeferredComponents(kvp, errors);
       case 'disable-swift-package-manager':
-        if (yamlValue is! bool) {
-          errors.add(
-            'Expected "$yamlKey" to be a bool, but got $yamlValue (${yamlValue.runtimeType}).',
-          );
-        }
+        errors.add(
+          // ignore: missing_whitespace_between_adjacent_strings
```

Comment:
Is this `ignore` needed? Seems to me all the string have `\n` between them

Reply:
That was a mistake, I thought I hit auto-fix not auto-ignore. Done.